src/index.tsx

```diff
@@ -358,6 +360,43 @@ function deleteSelectedElements() {
   }
 }
 
+function save() {
+  if (elements && elements.length > 0) {
+    const items = [...elements];
```

Comment:
Note that this doesn't do a deep copy. So you're still mutating the original elements two lines below